integr8ly/installation
# mobile playbooks

## Additional Information


Refactors mobile installation into its own platbook(s).

## Verification Steps


* run the install playbook
* check if any mobile specific product was installed
* run the install_mobile playbook
* check if all mobile products were properly installed
* run the install_mobile playbook and check if something went wrong (if mdc login stopped working for example)
* run the uninstall_mobile playbook and check if all mobile products were properly removed
* run the uninstall playbook

NOTE: a first time installation can also be achieved by running the usual install playbook adding the mobile flag: `-e mobile=true`

## Is an upgrade task required and are there additional steps needed to test this?
